scripts/add_to_bashrc.sh

```diff
@@ -14,5 +11,5 @@ To help you, we have already set the following environment variables:
 BUILDDIR = \"$BUILDDIR\"
 TOP_SRCDIR = \"$TOP_SRCDIR\"
```

Comment:
These additional assignments may be unnecessary as the variables are defined in the image. Are you able to access these variables within the dev container if these lines are removed?